### ISSUE TITLE
The instance crashes abnormally

### ISSUE THREAD

**Issue by boomballa** [dragonflydb/dragonfly#1783]
**Describe the bug**
I deployed a set of master-slave instances of dragonfly. The major versions of the system are consistent, both are debian11. However, the instance where the replica is located has always had crashes, while the instance where the master is located has no such problems. It has been running normally for 3 days Provide services. And there is no prompt in the log. This problem reappeared twice in two days. I am not sure whether it is caused by kernel incompatibility.

<img width="956" alt="image" src="https://github.com/dragonflydb/dragonfly/assets/8543659/6f44d01f-6952-43ea-a956-f0cc403963ca">

The start command is:
`dragonfly --flagfile=/dragonfly/dragonfly6379/dragonfly6379.conf &`

The configuration file is(Since my kernelban version is less than or equal to 5.10, I turned on the parameter: `--force_epoll` ):

```
--port=6379
--bind=xx.xx.xx.xx
--requirepass=xxxx
--masterauth=xxxx
--maxmemory=196gb
--dir=/dragonfly/dragonfly6379
--pidfile=/dragonfly/dragonfly6379/dragonfly6379.pid
--log_dir=/dragonfly/dragonfly6379
--version_check=true
--dbnum=1
--hz=100
--admin_port=16379
--admin_bind=xx.xx.xx.xx
--force_epoll=true
--epoll_file_threads=50
```

**Environment (please complete the following information):**
 - OS: [Debian GNU/Linux 11]
 - Kernel: 
  master:   Linux 5.10.0-20-amd64 x86_64
  replica:   Linux 5.10.0-22-amd64 x86_64
 - Dragonfly Version: [v1.8.0-7c99d2d1111e2636556ad8c2afad43c396906d01]

**Additional context**
I have checked the log, and there is no reason for the abnormal exit of any instance. 
<img width="398" alt="image" src="https://github.com/dragonflydb/dragonfly/assets/8543659/1bdab696-cc6e-410a-93e7-846fa4f6d5e7">
The last crash time yesterday was 2023-08-31 20:10, and the last print time of the log was 2023/08/31 11:44, so when the instance crashed, there was no log output.

In addition, after starting the replica instance today and re-establishing the master-slave relationship, individual errors were reported in the logs. It is not clear what caused the problem, but it did not affect the relationship between the master-slave instance and the operation of the instance.

Running duration (h:mm:ss): 0:00:00
Log line format: [IWEF]yyyymmdd hh:mm:ss.uuuuuu threadid file:line] msg
I20230901 14:45:36.193902 1052850 init.cc:69] dragonfly running in opt mode.
I20230901 14:45:36.194146 1052850 dfly_main.cc:666] Starting dragonfly df-v1.8.0-7c99d2d1111e2636556ad8c2afad43c396906d01
I20230901 14:45:36.194460 1052850 dfly_main.cc:722] Max memory limit is: 196.00GiB
I20230901 14:45:36.201424 1052850 proactor_pool.cc:147] Running 56 io threads
I20230901 14:45:36.202278 1052850 dfly_main.cc:397] Listening on admin socket xx.xx.xx.xx:16379
I20230901 14:45:36.304236 1052850 server_family.cc:171] Data directory is "/dragonfly/dragonfly6379"
I20230901 14:45:36.307174 1052854 listener_interface.cc:97] sock[116] AcceptServer - listening on port 16379
I20230901 14:45:36.307405 1052857 listener_interface.cc:97] sock[118] AcceptServer - listening on port 6379
I20230901 14:45:46.391122 1052853 server_family.cc:2008] Replicating xx.xx.xx.xx:6379
I20230901 14:45:46.419191 1052853 protocol_client.cc:219] Resetting endpoint! xx.xx.xx.xx, 6379
E20230901 14:45:46.467530 1052853 server_family.cc:1488] **Subcommand KILL not supported**
E20230901 14:45:46.467924 1052853 server_family.cc:1488] **Subcommand KILL not supported**
I20230901 14:45:46.583073 1052853 replica.cc:471] Started full sync with xx.xx.xx.xx:6379
I20230901 14:47:05.484570 1052853 replica.cc:490] Full sync finished in 1.32 min
I20230901 14:47:05.510784 1052853 replica.cc:566] Transitioned into stable sync
W20230901 14:47:46.586427 1052858 dfly_main.cc:116] **Remote version - connection error [version.dragonflydb.io:443] : Connection timed out**

**Comment by romange** [comment]
@boomballa  unfortunately this information is not enough to understand the reason for the crash.
the are also instance logs (journalctl, /var/log/syslog) and also systemd logs that could help understanding. Otherwise, we can not help here.

**Comment by boomballa** [comment]
> @boomballa unfortunately this information is not enough to understand the reason for the crash. the are also instance logs (journalctl, /var/log/syslog) and also systemd logs that could help understanding. Otherwise, we can not help here.

@romange Ok, I checked these logs before, but unfortunately I didn’t find much helpful information. This time I added nohup before starting the command, and there is no crash now, but the strange thing is that I also Nohup was not added, but there has been no crash problem.

In short, I will add it when I start it later, and I will continue to observe it. If I find helpful information later, I will give feedback.

thanks for your help .
